### ISSUE TITLE
SqlServerPermission: Add missing server permissions

### ISSUE THREAD

**Issue by XabiBeltza** [PowerShell/SqlServerDsc#1118]
#### Details of the scenario you tried and the problem that is occurring
It should be possible to add any server permissions, not just the current four. Specifically, it would help to add the permissions 'VIEW ANY DATABASE' and 'VIEW ANY DEFINITION', which are required to configure an SCOM SQL MP monitor account (https://blogs.msdn.microsoft.com/tysonpaul/2016/04/09/sql-run-as-account-configuration-for-scom-made-easy/)

#### The DSC configuration that is using the resource (as detailed as possible)
N/A

#### Version of the operating system and PowerShell the target node is running
Windows Server 2016, inbox powershell 

#### SQL Server edition and version the target node is running
Edition  : Standard Edition
Version  : 13.1.4001.0
Language : 1033

#### What SQL Server PowerShell modules, and which version, are present on the target node.                                                                               
xSQLServer 9.1.0.0 C:\Program Files\WindowsPowerShell\Modules\xSQLServer\9.1.0.0\xSQLServer.psd1            
xSQLServer 9.0.0.0 C:\Program Files\WindowsPowerShell\Modules\xSQLServer\9.0.0.0\xSQLServer.psd1            
SQLPS      1.0     C:\Program Files (x86)\Microsoft SQL Server\130\Tools\PowerShell\Modules\SQLPS\SQLPS.psd1

#### Version of the DSC module you're using, or write 'dev' if you're using current dev branch
SqlServerDsc 11.1.0.0 C:\Program Files\WindowsPowerShell\Modules\SqlServerDsc\11.1.0.0\SqlServerDsc.psd1

**Comment by johlju** [comment]
@XabiBeltza I agree. Added help wanted so that someone in the community can run with this one.
